app.js

```diff
@@ -76,6 +98,48 @@ app.get('/bus-positions', async function (req, res)
     }
 });
 
+app.get('/alerts', async function (req, res)
+{
+    let response = await fetch('https://glphprdtmgtfs.glphtrpcloud.com/tmgtfsrealtimewebservice/alert/alerts.pb');
+    if (response.ok)
+    {
+        let responseData = await response.arrayBuffer();
+        
+        let root = protobuf.loadSync('gtfs-realtime.proto');
+            
+        let FeedMessage = root.lookupType("transit_realtime.FeedMessage");

```

Comment:
Can we make these two variables global since they're shared between this and the bus positions endpoint?